### PR TITLE
[PORT]: ports the duke nukem announcer from monkestation

### PR DESCRIPTION
**Pull request by Yarinoi**
hail to the king baby
thanks to the people behind monkestation for doing most of the hard work so that i can steal it i guess

# Document the changes in your pull request

exactly what the title says. this pr ports the duke nukem announcer from monkestation to our humble yogstation, with a weight of 0.05 because duke nukem should not be taken in excess.

currently need to do:

- [x] THE BARE GODDAMN MINIMUM
files uncursed with the help of azzzertyy, thanks
- [x] MAKING SURE THE PR COMPILES
i think the tests broke cause the pr doesnt touch mapping at all
- [x] TESTING AND FIXING ISSUES
i'll do this soon trust

# Why is this good for the game?
more variety in announcers, another rare funny moment/funny bus opportunity, and in general, anything is made better with more duke nukem.

# Testing
i forgot to edit this, testing done, duke nuke nukes dukes

# Spriting
no

# Wiki Documentation
update station traits, if thats recorded anywhere, i guess?

# Changelog

:cl:  
rscadd: DUKE NUKEM IS HERE, READY TO TERRORIZE THOSE ALIEN BASTARDS OVER THE ANNOUNCEMENT SYSTEM.
/:cl:

### CODE REVIEW

**Comment by Yarinoi** [comment]
technically done, but one of the welcome duke lines mentions monkestation, and i cant play them because i had to uninstall something in order to unhaunt the voice lines, so... can someone tell me which one that is :)

File: code/datums/station_traits/neutral_traits.dm [hunk 1]
```diff
@@ -54,12 +55,27 @@
...
 /datum/station_trait/announcement_medbot/New()
 	. = ..()
 	SSstation.announcer = /datum/centcom_announcer/medbot
 
+/datum/station_trait/announcement_duke
+	name = "Announcement Duke"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 0.05
+	show_in_report = TRUE
+	report_message = "All our announcement systems are down and our interns are on strike, so we hired some guy off the street."
```
Comment (AMyriad) [inline code/datums/station_traits/neutral_traits.dm:70]: lol

**Comment by JamieD1** [comment]
> technically done, but one of the welcome duke lines mentions monkestation, and i cant play them because i had to uninstall something in order to unhaunt the voice lines, so... can someone tell me which one that is :)

Yo, this still being worked on?

**Comment by Yarinoi** [comment]
> > technically done, but one of the welcome duke lines mentions monkestation, and i cant play them because i had to uninstall something in order to unhaunt the voice lines, so... can someone tell me which one that is :)
> 
> Yo, this still being worked on?

i mean everything is pretty much done barring that one voice line, i'll quickly look through em

**Comment by Yarinoi** [comment]
it turns out what i thought was one voiceline was actually 2 out of the three welcome voicelines. will the crew have to live with a reference to another station or should i do something about it (remove them, cut out the reference to monkestation in the audio itself)?

**Comment by Yarinoi** [comment]
> it turns out what i thought was one voiceline was actually 2 out of the three welcome voicelines. will the crew have to live with a reference to another station or should i do something about it (remove them, cut out the reference to monkestation in the audio itself)?

mind changed, the people will survive. you can merge it

**Comment by JamieD1** [comment]
> > it turns out what i thought was one voiceline was actually 2 out of the three welcome voicelines. will the crew have to live with a reference to another station or should i do something about it (remove them, cut out the reference to monkestation in the audio itself)?
> 
> mind changed, the people will survive. you can merge it

Can we just change it to Yogstation? Im happy to change them if you cant.

**Comment by Yarinoi** [comment]
> > > it turns out what i thought was one voiceline was actually 2 out of the three welcome voicelines. will the crew have to live with a reference to another station or should i do something about it (remove them, cut out the reference to monkestation in the audio itself)?
> > 
> > 
> > mind changed, the people will survive. you can merge it
> 
> Can we just change it to Yogstation? Im happy to change them if you cant.

i dont see how we'd change the actual audio itself so unless you're willing to commission gianni matragrano or know anyone who can do a good duke nukem impression.. i unno

**Comment by AMyriad** [comment]
I think it might be best to leave out a voice line that has nothing to do with our server
it'd be about as awkward as the "welcome to TG station" you hear when arriving

**Review by ToasterBiome — APPROVED** [review]

**Pull request merged**